slidez-fe-core/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { HashRouter, Switch } from 'react-router-dom'
 import PublicRoute from './common/routes/public-route'
 import Dashboard from './pages/dashboard/Dashboard'
-import PrivateRoute from './common/routes/PrivateRoute'
+import PrivateRoute from './common/routes/private-route'
```

Comment:
Name files from big letter if we make so with all files

Reply:
Majority of files on frontend named in-this-style, do I have to rename them as well?

Reply:
I think we name components InThisStyle
Helpers and services can be in-this-style

Reply:
Private and Public route are components